src/parents.js

```diff
@@ -25,7 +25,8 @@ export function setPath (node, parentPath, { force } = {}) {
  * @returns {Node | null | undefined} The parent node, or undefined if the node's parent is unknown
  */
 export function getParent (node) {
-	return parentMap.get(node)?.node;
+	const path = getPath.call(this, node);
```

Comment:
I guess this is to centralize the logic of how paths are retrieved, but it doesn't seem worth it?

Reply:
Yes to centralize, but also because in `getPath` it's doing the work to check `this.root` if no parent pointers are set, so if one of them does it, than the other should too